account-management-api/src/main/java/uk/gov/di/accountmanagement/entity/UpdateEmailRequest.java

```diff
@@ -4,38 +4,17 @@
 import com.google.gson.annotations.SerializedName;
 import uk.gov.di.authentication.shared.validation.Required;
 
-public class UpdateEmailRequest {
-
-    @Expose
-    @SerializedName("existingEmailAddress")
-    @Required
-    private String existingEmailAddress;
-
-    @Expose
-    @SerializedName("replacementEmailAddress")
-    @Required
-    private String replacementEmailAddress;
-
-    @Expose @Required private String otp;
-
-    public UpdateEmailRequest() {}
+public record UpdateEmailRequest(
+        @Expose @SerializedName("existingEmailAddress") @Required String existingEmailAddress,
+        @Expose @SerializedName("replacementEmailAddress") @Required String replacementEmailAddress,
+        @Expose @Required String otp) {
 
     public UpdateEmailRequest(
             String existingEmailAddress, String replacementEmailAddress, String otp) {
-        this.existingEmailAddress = existingEmailAddress.toLowerCase();
-        this.replacementEmailAddress = replacementEmailAddress.toLowerCase();
+        this.existingEmailAddress =
+                existingEmailAddress != null ? existingEmailAddress.toLowerCase() : null;
```

Comment:
Is there any particular reason we added this null check in here? (Thoughts being I'd rather not have assumptions that things could be null, I'd rather use optionals in all nullable cases)